sdks/java/javadoc/ant.xml

```diff
@@ -70,19 +81,15 @@
       <link href="http://junit.sourceforge.net/javadoc/"
             offline="true" packageListLoc="junit-docs"/>
       
-      <excludepackage name="org.apache.beam.examples"/>
-      <excludepackage name="org.apache.beam.examples.common"/>
-      <excludepackage name="org.apache.beam.examples.cookbook"/>
-      <excludepackage name="org.apache.beam.examples.complete"/>
-      <excludepackage name="org.apache.beam.examples.complete.game"/>
-      <excludepackage name="org.apache.beam.examples.complete.game.injector"/>
-      <excludepackage name="org.apache.beam.examples.complete.game.utils"/>
-      <excludepackage name="org.apache.beam.runners.apex.translation"/>
-      <excludepackage name="org.apache.beam.runners.dataflow.internal"/>
-      <excludepackage name="org.apache.beam.runners.flink.examples"/>
-      <excludepackage name="org.apache.beam.runners.flink.translation"/>
-      <excludepackage name="org.apache.beam.runners.spark.examples"/>
-      <excludepackage name="org.apache.beam.runners.spark.translation"/>
+      <excludepackage name="org.apache.beam.examples.*"/>
```

Comment:
confirming differences from orig: you believe this works recursively? I think the original author did not.